src/main/java/de/maxhenkel/camera/StorageInterface.java

```diff
@@ -0,0 +1,8 @@
+package de.maxhenkel.camera;
+
```

Comment:
Are there other interfaces in this project? What’s the naming convention?

Reply:
None